src/index.html

```diff
@@ -1,13 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          class="img"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        >
+      </div>
+
+      <div class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="card__id">
+          Код товара: 195434
+      </div>
+
+      <div class="card__rating">
+        <div class="rating">
+          <div class="rating__star rating__star--good"></div>
+          <div class="rating__star rating__star--good"></div>
+          <div class="rating__star rating__star--good"></div>
+          <div class="rating__star rating__star--good"></div>
+          <div class="rating__star rating__star--bad"></div>
+        </div>
+
+        <div class="reviews">
```

Comment:
```suggestion
        <div class="card__reviews">
```
Why would you create a separate block only for this small text field?